oslo/torch/nn/parallel/data_parallel/zero/heterogeneous_manager/chunk/chunk.py

```diff
@@ -0,0 +1,638 @@
+from dataclasses import dataclass
```

Comment:
Note: this does not change from the original code except the process_group.
https://github.com/hpcaitech/ColossalAI/blob/cd142fbefa964d62048a9bafb180322369ab89f8/colossalai/gemini/chunk/chunk.py#L58